modules/core/src/main/scala/org/tesselation/infrastructure/cluster/storage/ClusterStorage.scala

```diff
@@ -27,6 +27,9 @@ object ClusterStorage {
       def getPeers: F[Set[Peer]] =
         peers.keys.flatMap(_.map(peers(_).get).sequence).map(_.flatten.toSet)
 
+      def getPeer(id: PeerId): F[Option[Peer]] =
```

Comment:
It's not always useful. When you keep all your outputs like this, then fine. (like EitherT in our previous SnapshotService class). When it's for a single usage only, it's pointless to wrap it there into monad transformer, because you need to unwrap it anyway (because we're not using OptionT flows, but F only)

Reply:
Hm, ok let's keep it in `F` then.